turbinia/evidence.py

```diff
@@ -958,3 +958,38 @@ def _postprocess(self):
       # Unmount the container's filesystem
       mount_local.PostprocessUnmountPath(self._container_fs_path)
       self.state[EvidenceState.CONTAINER_MOUNTED] = False
+
+
+#TODO implement support for several ewf devices if there are more than one
+#inside the block path
+class EwfDisk(RawDisk):
+  """Evidence object for a EWF based evidence.
+
+  Attributes:
+    device_path (str): Path to the mounted loop device.
+    ewf_path (str): Path to mounted EWF image.
+    block_path (str): Path to EWF mount directory.
+  """
+  POSSIBLE_STATES = [EvidenceState.ATTACHED, EvidenceState.MOUNTED]
+
+  def __init__(self, *args, **kwargs):
+    """Initialization for EWF evidence object."""
+    self.device_path = None
+    self.ewf_path = None
+    self.block_path = None
+    super(EwfDisk, self).__init__(*args, **kwargs)
+
+  def _preprocess(self, _, required_states):
+    if EvidenceState.ATTACHED in required_states:
+      self.block_path = mount_local.PreprocessMountEwfDisk(self.source_path)
```

Comment:
Since IIUC this is actually the mount point for where the block devices go, can we rename this to `ewf_mount_path` or similar just so it's a little clearer what is happening here?

Reply:
thanks